NEWS.md

```diff
@@ -1,89 +1,109 @@
+
+*   In non-user-facing internal code (specifically in the `aes()` function and in
+    the `aesthetics` argument of scale functions), ggplot2 now always uses the British
+    spelling of aesthetics containing the word "colour". When users specify a "color"
+    aesthetic it is automatically renamed to "colour". This principle is also applied
+    to non-standard aesthetics that contain the word "color". For example, "point_color"
+    is renamed to "point_colour". This convention makes it easier to support both
+    British and American spelling for novel non-standard aesthetics, but it may require
+    some adjustment for packages that have previously introduced non-standard color
+    aesthetics using American spelling. A new function `standardise_aes_names()` is
+    provided in case extension writers need to perform this renaming in their own code
+    (@clauswilke, #2649).
+
+*   Functions that generate other functions (closures) now force the arguments that are
+    used from the generated functions, to avoid hard-to-catch errors. This may affect
+    some users of manual scales (such as `scale_colour_manual()`, `scale_fill_manual()`,
+    etc.) who depend on incorrect behavior (@krlmlr, #2807).
     
 *   `Coord` objects now have a function `backtransform_range()` that returns the
     panel range in data coordinates. This change may affect developers of custom coords,
     who now should implement this function. It may also affect developers of custom
     geoms that use the `range()` function. In some applications, `backtransform_range()`
-    may be more appropriate.
-    (@clauswilke, #2821).
+    may be more appropriate (@clauswilke, #2821).
 
-*   `geom_sf()` now respects `lineend`, `linejoin`, and `linemitre` parameters 
-    for lines and polygons (@alistaire47, #2826)
 
-*   `geom_hline()`, `geom_vline()`, and `geom_abline()` now work properly
-    with `coord_trans()` (@clauswilke, #2149, #2812).
+## New features
+
+*   `coord_sf()` has much improved customization of axis tick labels. Labels can now
+    be set manually, and there are two new parameters, `label_graticule` and
+    `label_axes`, that can be used to specify which graticules to label on which side
+    of the plot (@clauswilke, #2846, #2857, #2881).
+    
+*   Two new geoms `geom_sf_label()` and `geom_sf_text()` can draw labels and text
+    on sf objects. Under the hood, a new `stat_sf_coordinates()` calculates the
+    x and y coordinates from the coordinates of the sf geometries. You can customize
+    the calculation method via `fun.geometry` argument (@yutannihilation, #2761).
+    
+
+## Minor improvements and fixes
 
 *   `benchplot()` now uses tidy evaluation (@dpseidel, #2699).
 
+*   The error message in `compute_aesthetics()` now provides the names of only
```

Comment:
Awk word ordering here, maybe change to: "now only provides the names of aesthetics…" or "now returns the names only for aesthetics…"

Reply:
Ok, thanks!